src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectOptions.cs

```diff
@@ -241,5 +241,12 @@ public new IOpenIdConnectEvents Events
         /// This is disabled by default.
         /// </summary>
         public bool SkipUnrecognizedRequests { get; set; } = false;
+
+        /// <summary>
+        /// Indicates whether telemetry should be disabled. When this feature is disabled,
```

Comment:
Perhaps change the second sentence to be worded with positives instead of the double-negative? E.g. "When this feature is enabled, blah blah is sent to the blah blah". I just think it's a bit easier to read that way.